src/datagrid/DataGrid.tsx

```diff
@@ -268,12 +277,12 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     }
 
     componentDidUpdate(prevProps: DataGridProps) {
-        const {rows, columns, pagination} = this.props;
+        const {rows, columns, pagination, footer} = this.props;
```

Comment:
Do you need `footer`?  Looks like it's never used.

Reply:
done